.eslintrc.js

```diff
@@ -92,6 +100,19 @@ module.exports = {
     'import/no-webpack-loader-syntax': 'error',
     'import/unambiguous': 'error',
 
+    'import-helpers/order-imports': [
+      'error',
+      {
+        newlinesBetween: 'always',
+        groups: [
+          '/^(assert|async_hooks|buffer|child_process|cluster|console|constants|crypto|dgram|dns|domain|events|fs|http|http2|https|inspector|module|net|os|path|perf_hooks|process|punycode|querystring|readline|repl|stream|string_decoder|timers|tls|trace_events|tty|url|util|v8|vm|zli)/',
```

Comment:
Where does this list come from?

Reply:
https://github.com/Tibfib/eslint-plugin-import-helpers/blob/master/docs/rules/order-imports.md#upgrading-from-v014-to-v1